docs/HowToDevelopThingsUsingSTDK.md

```diff
@@ -0,0 +1,256 @@
+# How to develop things using STDK based TizenRT   
+
+STDK is one of Develop Kit(STDK) for Smart Things Eco-system.   
+This guide shows an example how to enable STDK and develop things.   
+
+So now, it will explain how to develop Led Things of SmartThings and Control it through SmartThings Mobile app.   
+
```

Comment:
I think it would be useful to have a outline for the document so that readers can jump directly to the part they are interested in. Please check https://github.com/Samsung/TizenRT/blob/master/docs/HowToUseSmartFS.md, to see how they do it.

Reply:
Has Changed.